origin-discovery/src/listener/handler_identity.js

```diff
@@ -266,6 +266,10 @@ class IdentityEventHandler {
   emailWebhookEnabled() {
     return true
   }
+
+  gcloudPubsubEnabled() {
+    return false
```

Comment:
Identity data is stored in IPFS. So shouldn't we publish that IPFS hash to pubsub ?

Reply:
Ah... Perhaps the Google Cloud Function rohit implemented does not support extracting IPFS hash from identity data... Let's add a TODO or file a separate issue to keep track of this ?

Reply:
Yep was going to do that as a separate PR. Also the unpinning if identity we discussed. Will file a separate issue.